src/main/java/foodwhere/model/review/ReviewBuilder.java

```diff
@@ -30,7 +30,7 @@ public ReviewBuilder() {
         name = new Name(DEFAULT_NAME);
         date = new Date(DEFAULT_DATE);
         content = new Content(DEFAULT_CONTENT);
-        rating = new Rating(DEFAULT_RATING);
+        rating = new Rating(String.valueOf(DEFAULT_RATING));
```

Comment:
Perhaps we can store DEFAULT_RATING as a String?

Reply:
Resolved in latest commit